server/blob/blob-cassandra/src/test/java/org/apache/james/blob/cassandra/cache/CachedBlobStoreTest.java

```diff
@@ -314,4 +314,17 @@ public void readBlobstoreCachedShouldCountWhenMissed() {
         assertThat(metricFactory.countFor(BLOBSTORE_CACHED_MISS_COUNT_METRIC_NAME))
             .isEqualTo(1);
     }
+
+    @Test
+    public void readBlobStoreCacheShouldNotCountWhenReadNonExistedBlob() {
```

Comment:
Do we record latencies when the blob do not exist? (Can you write a test about it?)

Reply:
The test name likely should be `readBlobStoreCacheShouldNotCountWhenReadNonExist**ing**Blob`

Reply:
> Do we record latencies when the blob do not exist? (Can you write a test about it?)

Yes, i will add for this case.

> The test name likely should be `readBlobStoreCacheShouldNotCountWhenReadNonExist**ing**Blob`

sound a better name

Reply:
I found a name shorter like : `metricShouldNotWorkWhenReadNonExistingBlob`